index.js

```diff
@@ -15,6 +15,7 @@ export default function entrypoint(options = {}) {
     useCloudTracing = false,
     dependencies = {},
     nodejsRuntime = 18,
+    defaultYamlFile = "app.yaml"
```

Comment:
I think it would make more sense to name the parameter something like 'appYamlFile' and not default, since the config overrides the default (which is app.yaml)